articles/tech-resources/enabling-multiple-custom-tracking-domains.md

```diff
@@ -47,6 +47,8 @@ To configure one or more custom tracking domains through the SparkPost user inte
     ![](media/enabling-multiple-custom-tracking-domains/xU06kJ2lac6YGEIi8KlReq8OaF1kfX.png)
 1. Complete the Sending Domain Name for each sending domain and select the Linked Tracking Domain you want to associate with it.
 1. Click the Add Domain or _Update Tracking Domain_ button to complete the association.
+Please note in order to associate a tracking domain with a sending doamin you need to make you have verified the trakcing domain first. 
```

Comment:
- Lets separate your additions from the list item above. Add a blank line above your text to do that.
 - 2 typos: `trakcing` -> `tracking`.
- Missing word: `make you` -> `make sure you`
- Please link to the tracking domain part of the app in your text. Its this page: https://app.sparkpost.com/account/tracking-domains
 - How about `...on the right hand side of your tracking domain`.

(I recommend using the "View" button on your pull request ([this page](https://github.com/SparkPost/support-docs/pull/187/files)) so you can see how your changes look.)